pkg/cmd/infra/deployer/doc.go

```diff
@@ -1,3 +1,3 @@
 // Package cmd contains various interface implementations for command-line tools
 // associated with deploymentconfigs.
-package cmd
+package deployer
```

Comment:
why is it moving to deployer? is that just a leftover from the change?

Reply:
that is goland not being that smart as i thought :-) good catch